pd-server/main.go

```diff
@@ -21,6 +21,7 @@ var (
 	leaderLease   = flag.Int64("lease", 3, "Leader lease time (second)")
 	logLevel      = flag.String("L", "debug", "log level: info, debug, warn, error, fatal")
 	pprofAddr     = flag.String("pprof", ":6060", "pprof HTTP listening address")
+	clusterID     = flag.Uint64("cluster-id", 0, "Cluster ID")
```

Comment:
em, 1 seems strange, I prefer 0 if 1. 

We can log a warning if we use 0. Btw for most cases, we use different root path not "/pd", so it is ok.